client/genericclient/client.go

```diff
@@ -40,6 +41,7 @@ func NewClientWithServiceInfo(destService string, g generic.Generic, svcInfo *se
 	var options []client.Option
 	options = append(options, client.WithGeneric(g))
 	options = append(options, client.WithDestService(destService))
+	options = append(options, client.WithMetaHandler(transmeta.ClientTTHeaderHandler))
```

Comment:
It is not recommended to add this. The internal Kitex will invoke this function, and there will be conflicts with the ClientTTHeaderHandler.